bsip-0022.md

```diff
@@ -37,34 +37,202 @@ Currently within Bitshares DPOS system, an user's votes for committee/witness/pr
+
+### <div id="eos-vote-decay" /> Vote Decay on EOS
+
+Vote decay was [introduced into the EOS blockchain with Dawn 4.0](https://medium.com/@auroraeos/understanding-eos-vote-decay-e50c58b56658).  The maximum voting power of an account is a function of [the core tokens that are staked by that account](https://github.com/eosio/eos/eosio.system/blob/bfd1793032ed69bba8047b4807f692eaed2ed5e5/eosio.system/src/voting.cpp#L228).  This maximum effectively [step decays](https://github.com/eosio/eos/eosio.system/blob/bfd1793032ed69bba8047b4807f692eaed2ed5e5/eosio.system/src/voting.cpp#L122-126) every week [by 1.34%](https://www.eoscanada.com/en/how-is-your-vote-strength-calculated-on-eos) unless the account re-casts its votes.  The vote "decay" is technically implemented by [strengthening newer votes](https://github.com/eosio/eos/eosio.system/blob/bfd1793032ed69bba8047b4807f692eaed2ed5e5/eosio.system/src/voting.cpp#L122-126).  The age of the last casted vote [affects the voting power for all referendums](https://github.com/eosio/eos/eosio.system/blob/bfd1793032ed69bba8047b4807f692eaed2ed5e5/eosio.system/src/voting.cpp#L200-319) on the platform.  The decaying of votes for block producers [reduces the amount of pay](https://github.com/eosio/eos/eosio.system/blob/bfd1793032ed69bba8047b4807f692eaed2ed5e5/eosio.system/src/producer_pay.cpp#L153-171) received by [standby block producers](https://medium.com/eostribe/how-eos-block-producers-are-paid-7b2a1216eb2b).
+
+### <div id="peerplays-vote-decay" /> Vote Decay on Peerplays
+
+A [proposal for Peerplays](https://github.com/peerplays-network/pips/blob/master/pip-0002.md) introduced vote decay as an element of its Gamified Proof of Stake (GPOS).  Voters receive rewards from a portion of the operation fees and "rakes" that are collected by the blockchain.  The reward is contingent on [the amount of stake that is vested by the voter](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L1072-1080), and [the age of last casted vote by the voter](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L726-791).  The age of the last casted vote [also decays the voting power](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L1446-1447) for all referendums on the platform.
+
+
 # Specifications
 
-## Committee controlled variables
+## <div id="referendum-categories" /> Referendum Categories
+
+BitShares has four categories of refereundums that core token holders may vote on: witnesses, committee members, worker proposals, and proxy delegation.  Prior to this proposal, any vote within any category applies until changed or until the proxy assignment had been assigned.  The removal of proxy assignment re-instates a core token holders' prior voting selection for witnesses, commitee members.
```

Comment:
Typo: refere*u*ndums

Reply:
> Typo: refere_u_ndums ... commit_t_ee

A correction will be made


> I don't think proxy delegation is a Category of its own. It's not like we're voting _for_ proxies, we're voting _via_ proxies.

@pmconrad Is your objection specifically about calling it a "referendum category"?  Or is it about decaying the delegation for a proxy?


> Removal of proxy reinstates prior votes also for workers.

A correction will be made

Reply:
> I don't think proxy delegation is a Category of its own. It's not like we're voting _for_ proxies, we're voting _via_ proxies.

After some thought, I have come to agree.  The draft has been updated to make this distinction.